VARIABLE.md

```diff
@@ -0,0 +1,128 @@
+# VARIABLES
+
+Variable is used to label and store data in computer memory.
+Variable can update information stored in variable,
+reference or get information stored in a variable.
+
+Its important to distinguish that variable are not value, they contain value and represent them.
+
+`Let` and `const` are new keyword to declare a variable, introduced in `ES6`. Prior to that programmers can declare variable with only the `var` keyword. example:
+
+```js
+var food = 'pizza';
+console.log(food); // output: pizza
+```
+
+`let` keyword signal that values can be reassigned a different value e.g
+
+```js
+let myName = 'Azeez';
+console.log(myName); //output: Azeez
+myName = 'Rotimi';
+console.log(myName); //output: Rotimi
+
+```
+
+If the `let` variable is declared without a value, the variable will be automatically initialized with a value of `undefined` example:
+
+```js
+let myAge;
+console.log(myAge); // output: undefined.
+```
+
+
+The `let` keyword is similar to using the `var` keyword because, anything that can be done with `var` can be done with `let`.
```

Comment:
The statement here may not be entirely true, I would recommend investigating the difference between `var` and `let` more closely.

Reply:
I have made some findings, it has been restructured.